### ISSUE TITLE
compile failure with GHC 7.6 & GHC 7.8

### ISSUE THREAD

**Issue by hvr** [bgamari/pipes-lzma#1]
```
[1 of 1] Compiling Pipes.Lzma       ( src/Pipes/Lzma.hs, dist/dist-sandbox-4d6c05da/build/Pipes/Lzma.o )

src/Pipes/Lzma.hs:59:44:
    Not in scope: `mempty'
    Perhaps you meant `BS.empty' (imported from Data.ByteString)

src/Pipes/Lzma.hs:92:44:
    Not in scope: `mempty'
    Perhaps you meant `BS.empty' (imported from Data.ByteString)
xcabal: Error: some packages failed to install:
pipes-lzma-0.1.0.0 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by hvr** [comment]
May I suggest to give https://github.com/hvr/multi-ghc-travis/blob/master/make_travis_yml_2.hs a try? :-)

**Comment by hvr** [comment]
fyi, https://hackage.haskell.org/package/pipes-lzma-0.1.0.0/revisions/

**Comment by bgamari** [comment]
Fixed in 0.1.1.0.
